practice-exam/practice-exam-5.md

```diff
@@ -372,6 +375,8 @@ If this practice exam has been helpful to you please share it with others and re
 
     <details markdown=1><summary markdown='span'>Answer</summary>
       Correct answer: B
+      `Shouldn't this be AWS WAF. Therefore Correct answer: B`
```

Comment:
Please check https://github.com/kananinirav/AWS-Certified-Cloud-Practitioner-Notes/issues/207 for more details